.github/workflows/examples.yml

```
@@ -0,0 +1,128 @@
+      - "master"
+      - "maintenance/.+"
+  pull_request:
+    branches:
+      - "master"
+      - "maintenance/.+"
+  schedule:
+    # Nightly tests run on master by default:
+    #   Scheduled workflows run on the latest commit on the default or base branch.
+    #   (from https://help.github.com/en/actions/reference/events-that-trigger-workflows#scheduled-events-schedule)
+    - cron: "0 0 * * *"
+
+jobs:
+  test:
+    if: (github.event_name == 'schedule' && github.repository == 'openforcefield/openff-toolkit') || (github.event_name != 'schedule')
+    name: ${{ matrix.os }}, Python ${{ matrix.cfg.python-version }}, RDKit=${{ matrix.cfg.rdkit }}, OpenEye=${{ matrix.cfg.openeye }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          - ubuntu-latest
+          - macos-latest
+        cfg:
+          - python-version: 3.7
+            rdkit: "true"
+            openeye: "false"
+
+          - python-version: 3.7
+            rdkit: "false"
+            openeye: "true"
+
+
+    env:
+      CI_OS: ${{ matrix.os }}
+      RDKIT: ${{ matrix.cfg.rdkit }}
+      OPENEYE: ${{ matrix.cfg.openeye }}
+      PYVER: ${{ matrix.cfg.python-version }}
+      OE_LICENSE: ${{ github.workspace }}/oe_license.txt
+      PACKAGE: openff
+      PYTEST_ARGS: -r fE --tb=short --cov=openff --cov-config=setup.cfg --cov-append --cov-report=xml
+      NB_ARGS: -v --nbval-lax --ignore=examples/deprecated
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 2
+
+      - uses: conda-incubator/setup-miniconda@v2
+        name: Install only RDKit
+        if: ${{ matrix.cfg.rdkit == 'TRUE' && matrix.cfg.openeye == 'FALSE' }}
+        with:
+          python-version: ${{ matrix.cfg.python-version }}
+          activate-environment: test
+          environment-file: devtools/conda-envs/rdkit.yaml
+          auto-activate-base: false
+      - uses: conda-incubator/setup-miniconda@v2
+        name: Install with OpenEye toolkits
+        if: ${{ matrix.cfg.rdkit == 'FALSE' && matrix.cfg.openeye == 'TRUE' }}
+        with:
+          python-version: ${{ matrix.cfg.python-version }}
+          activate-environment: test
+          environment-file: devtools/conda-envs/openeye.yaml
+          auto-activate-base: false
+
+      - name: Install additional example dependencies
+        shell: bash -l {0}
+        run: |
+          conda env update --file examples/environment.yaml --name test
+
+      - name: Additional info about the build
+        shell: bash -l {0}
+        run: |
+          uname -a
+          df -h
+          ulimit -a
+
+      - name: Make oe_license.txt file from GH org secret "OE_LICENSE"
+        shell: bash
+        env:
+          OE_LICENSE_TEXT: ${{ secrets.OE_LICENSE }}
+        run: |
+          echo "${OE_LICENSE_TEXT}" > ${OE_LICENSE}
+
+      - name: Install package
+        shell: bash -l {0}
+        run: |
+          python setup.py develop --no-deps
```

Because openmmforcefields would have pulled in the latest _packaged_ version of openff-toolkit, let's delete it here, to ensure that only the current codebase is being tested. 

```suggestion
          conda remove --force openff-toolkit
          python setup.py develop --no-deps
```

---

Done in 0ac1b27